src/Summoner/Project.hs

```diff
@@ -51,12 +49,12 @@ generateProject projectName Config{..} = do
     email       <- queryDef "Maintainer e-mail: " cEmail
     putText categoryText
     category <- query "Category: "
-    license  <- choose "License: " $ map unLicense $ ordNub (cLicense : licenseNames)
+    license  <- choose "License: " $ map showLicense universe
```

Comment:
I guess the logic here changed a bit. 
We want in this list to have the default license (or the chosen one) to come first as it will be shown in brackets like `[MIT]/BSD-3/...`.

Reply:
It show like `[MIT]/BSD-3/...` still when summoner asks about license.

Reply:
@willbasky Yeah, but when you try it with `.summoner.toml` from our org, it should show `MPL-2.0`. But it shows `MIT`. It shows MIT now, because it's the first constructor. And `universe` starts with `minBound`

* https://github.com/kowainik/org/blob/master/.summoner.toml